packages/ruleset/test/operation-id-naming-convention.test.js

```diff
@@ -12,24 +12,186 @@ describe('Spectral rule: operation-id-naming-convention', () => {
       const results = await testRule(ruleId, rule, rootDocument);
       expect(results).toHaveLength(0);
     });
-  });
 
-  describe('Should yield errors', () => {
-    it('no path param, post', async () => {
+    it('path has multiple path params, get', async () => {
```

Comment:
Re-organized the tests so they are grouped by http method, and added a bunch of tests for some previously untested scenarios.